src/plugins/template/backend/CMakeLists.txt

```diff
@@ -2,19 +2,11 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-set (SRC
-    backend.cpp
-    backend.hpp
-    cache.cpp
-    cache.hpp
-    executable.cpp
-    executable.hpp
-    performance_counter.hpp
-    pass/dyn_elimination.cpp
-    pass/dyn_elimination.hpp
-    pass/shape_relevance.cpp
-    pass/shape_relevance.hpp
-    int_backend.cpp int_executable.cpp evaluates_map.cpp
+file(GLOB SRC
+    "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp"
```

Comment:
Can we create a separate folder for operations?
`src/plugins/template/backend/ops/`